assets/javascripts/modules/toggle.js

```diff
@@ -112,7 +112,7 @@ var toggleEvent = function ($elem) {
       if (event.which === 1 || 32) {
         var $input = $(event.target)
```

Comment:
Outside the scope of this PR but it appears like this is a bad name `$input`, we are defining it as an input and then saying if its a label then make it an input! Should be `$element` IMO issue raise #750